README.md

```diff
@@ -14,27 +15,84 @@
 
 <!-- <KFD-DOCS> -->
 
-Furyctl is a command line interface tool to:
+`furyctl` is the command line companion for the Kubernetes Fury Distribution to manage the **full lifecycle** of your Kubernetes Fury clusters.
+<br/>
+<!-- line left blank -->
+> 💡 Learn more about the Kubernetes Fury Distribution in the [official site](https://kubernetesfury.com).
+<!-- line left blank -->
+> **Warning** you are viewing the readme for furyctl next generation (`furyctl-ng` for short).
+>
+> `furyctl-ng` is in `alpha` status and currently supports EKS-based clusters only.
 
-- create and manage Fury clusters on AWS
-- download and manage the Kubernetes Fury Distribution (KFD) modules
+## Installation
 
-<br/>
+### Installation from source
 
-![Furyctl usage](docs/assets/furyctl.gif)
+Prerequisites:
 
-> 💡 Learn more about the Kubernetes Fury Distribution in the [official site](https://kubernetesfury.com).
+- `make`
+- `go == v1.19`
+- `goreleaser == v1.11.4`
 
-## Installation
+> You can install `goreleaser` with the following command once you have Go in your system: 
+>
+> ```console
+> go install github.com/goreleaser/goreleaser@v1.11.4
+> ```
+
+To install `furyctl` from source, follow the next steps:
+
+1. Clone the repository:
+
+<!-- FIXME: remove the branch swithing in the future -->
+```console
+git clone git@github.com:sighupio/furyctl.git
+# cd into the cloned repository
+cd furyctl
+# Switch to the branch for the `furyctl-ng-alpha1` version
+git switch furyctl-ng-alpha1
+```
+
+2. Build the binaries by running the following command:
+
+```console
+make build
+```
+
+3. You will find the binaries for Linux, Darwin (macOS) and Windows for your current architecture inside the `dist` folder:
+
+```console
+$ tree dist/furyctl_*/
+dist/furyctl_darwin_amd64_v1
+└── furyctl
+dist/furyctl_linux_amd64_v1
+└── furyctl
+dist/furyctl_windows_amd64_v1
+└── furyctl.exe
+```
+
+4. Check that the binary is working as expected:
+
+> **Note** replace `darwin` with your OS and `amd64` with your architecture in the following commands.
 
-### Installation from binaries
+```console
+./dist/furyctl_darwin_amd64_v1/furyctl version
+```
+
+5. (optional) move the binary to your `bin` folder, in macOS:
+
+```console
+sudo mv ./dist/furyctl_darwin_amd64_v1/furyctl /usr/local/bin/furyctl
+```
+
+### Installation from binaries (not available yet for `furyctl-ng`)
```

Comment:
I would remove everything that is not inherent to the new furyctl version

Reply:
They will be relevant once we figure out how to manage releases in parallel or when we fully switch to ng.

If we delete these sections, we'll have to add them back soon™. But if they generate confusion let's delete them.

IMHO is acceptable to have a section that is not YET relevant for an alpha release. There's a warning in the header saying that you are viewing docs for an alpha version.

Reply:
from my point of view, IF we need some information about the old one, we can use a link to the old README pointing to a tag, and call it a day

Reply:
This is not information about legacy, we'll use the same mechanism (and docs) for installing ng (bumping the version). These docs will be inherent to ng also, once we release it.

Reply:
OK